articles/fire-forests-inventories/components/utils.js

```diff
@@ -0,0 +1,17 @@
+export const getOption = (options) => {
+  return Object.keys(options)[
+    Object.keys(options)
+      .map((d) => options[d])
+      .indexOf(true)
+  ]
```

Comment:
tiny nit: I found this a little hard to read, maybe because of the explicit reference to object key index? I think the same thing can be done with something like
```suggestion
  return Object.keys(options).find(key => options[key])
```

Reply:
nice, much better